### ISSUE TITLE
NTR: hepatopulmonary fusion

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#4179]
_For new term requests, please provide us with the following information:_
### 1. Preferred term label
hepatopulmonary fusion

### 2. Synonyms
Hepatic pulmonary fusion

### 3. Textual definition (should be understandable even for non-specialists, please include a PubMed ID for relevant articles providing additional information about the suggestion)
Fusion of the liver with the lung.
PMID:28196822 

### 4. Parent term (use [HPO app](https://hpo.jax.org/app/), [HPO Browser](http://compbio.charite.de/hpoweb/showterm?id=HP:0000118) or [OLS](http://www.ebi.ac.uk/ols/ontologies/hp))
HP_0410042

### 5. Which diseases are characterized by this term ? (e.g. Orphanet- or OMIM-id)
Congenital diaphragmatic hernia

### 6. Your nano-attribution (ORCID-id or label, e.g. _HPO:probinson_ (organization:name))
https://orcid.org/0000-0001-5208-3432
